From afdf9d9124a96f847cef11a9e2a58d61aaafb8d1 Mon Sep 17 00:00:00 2001
From: leavs <leavs_o@126.com>
Date: Tue, 7 Nov 2017 15:37:44 +0800
Subject: [PATCH] Add Freescale Logo

---
 tools/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/Makefile b/tools/Makefile
index 4bbb153..f2ffd28 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -186,7 +186,7 @@ LOGO-$(CONFIG_VIDEO_LOGO) += $(LOGO_DATA_H)
 
 # Generic logo
 ifeq ($(LOGO_BMP),)
-LOGO_BMP= $(srctree)/$(src)/logos/denx.bmp
+LOGO_BMP= $(srctree)/$(src)/logos/freescale.bmp
 
 # Use board logo and fallback to vendor
 ifneq ($(wildcard $(srctree)/$(src)/logos/$(BOARD).bmp),)
-- 
1.9.1

